config/settings/local.py

```diff
@@ -13,11 +13,8 @@
     default="aHdCBMHXuxIxEhfRGFRp7Cp3N9CqEZEEAvwZVlBCazKExkEnzvVs4bYWC8Qqh9lg",
 )
 # https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = [
-    "localhost",
-    "0.0.0.0",
-    "127.0.0.1",
-]
+allowed_hosts = env("DJANGO_ALLOWED_HOSTS", default="localhost,0.0.0.0,127.0.0.1,*")
```

Comment:
The `*` will match anything. This should not be the default if `DJANGO_ALLOWED_HOSTS` is not set. Since this is the `local.py` configuration I'd change it to `.localhost,127.0.0.1,[::1]`

Reply:
I agree!